real_time/arduino_src/radio_buggy_mega/main.cpp

```diff
@@ -12,7 +12,7 @@
 #include "system_clock.h"
 #include "fingerprint.h"
 
-#define BAUD 9600
+#define BAUD 115200
```

Comment:
Do you think we coudl have the Baud rate defined in a common place between high and low level so that it will always be in sync?